.github/workflows/pr-opened.yml

```diff
@@ -1,22 +0,0 @@
-name: Pull Request Opened
-permissions:
-  pull-requests: write
-
-# only trigger on pull request closed events
-on:
-  pull_request_target:
-    types: [ opened ]
-
-jobs:
-  pr_open_job:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/github-script@d7906e4ad0b1822421a7e6a35d5ca353c962f410 # v6.4.1
-        with:
-          script: |
-            github.rest.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
-              body: "Reminder for the PR assignee: If this is a user-visible change, please update the changelog as part of the PR."
```

Comment:
Are you sure we want to remove that? That's a good reminder to update the changelog. I personally forgot a few times, but this message was helpful.